README.md

```diff
@@ -1,62 +1,46 @@
+React - Redux Project
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Bookstore
 
+> "Bookstore"
```

Comment:
Kindly give a proper description of your project. Remember it is super important to have a nice and descriptive README file.

Reply:
okay, noted! I just made the corrections